cfgov/jinja2/v1/browse-filterable/index.html

```diff
@@ -15,6 +16,30 @@
 {% block content_main %}
     <div class="block
                 block__flush-top">
+        {# TODO: Move notification to filter-list-control. #}
+        {{ notification.render() }}
+
+        {{ notification.render({
+            'type': 'warning'
+        }) }}
+
+        {{ notification.render({
+            'type': 'error'
+        }) }}
+
+        {{ notification.render({
+            'message': 'Custom text',
+            'explanation': 'With an explanation!'
+        }) }}
+
+        {{ notification.render({
+            'type': 'warning',
+            'message': 'Custom text',
+            'explanation': "And an explanation with a <a href='#'>link</a>" | safe
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/9102c4a290dfa97405c249c2165b547e7aa1df10